flink-connectors/flink-connector-cassandra/src/main/java/org/apache/flink/streaming/connectors/cassandra/CassandraSink.java

```diff
@@ -62,7 +62,7 @@ private CassandraSink(SingleOutputStreamOperator<IN> sink) {
     }
 
     private LegacySinkTransformation<IN> getSinkTransformation() {
-        return sink1.getTransformation();
+        return sink1.getLegacyTransformation();
```

Comment:
I couldn't find getLegacyTransformation in [DataStreamSink.java](https://github.com/apache/flink/blob/master/flink-streaming-java/src/main/java/org/apache/flink/streaming/api/datastream/DataStreamSink.java). I guess it may be related to `IN` or Cassandra? If so, could you refer me to the doc or source code for that function?